src/gui/processing/qgsprocessingmultipleselectiondialog.h

```diff
@@ -137,10 +142,16 @@ class GUI_EXPORT QgsProcessingMultipleSelectionPanelWidget : public QgsPanelWidg
 
     QList< QStandardItem * > currentItems();
 
-
     void populateList( const QVariantList &availableOptions, const QVariantList &selectedOptions );
 
     friend class TestProcessingGui;
+    bool mDragActive = false;
```

Comment:
Tracking this manually should not be required -- it's all handled for us by Qt.

Reply:
Thanks for the clarification, I wasn't sure what was needed to properly handle everything. I removed the dragLeaveEvent as I guess it doesn't matter much.